iamidentityv1/iam_identity_v1_examples_test.go

```diff
@@ -719,6 +719,140 @@ var _ = Describe(`IamIdentityV1 Examples Tests`, func() {
 			Expect(err).To(BeNil())
 			Expect(response.StatusCode).To(Equal(204))
 		})
+		It(`GetProfileIdentities request example`, func() {
+			fmt.Println("\nGetProfileIdentities() result:")
+			// begin-get_profile_identities
+
+			getProfileIdentitiesOptions := iamidentityv1.GetProfileIdentitiesOptions{
+				ProfileID: &profileId,
+			}
+
+			profileIdnetities, response, err := iamIdentityService.GetProfileIdentities(&getProfileIdentitiesOptions)
```

Comment:
Is this a manual change?  Not sure how else this variable could become misspelled :)
```suggestion
			profileIdentities, response, err := iamIdentityService.GetProfileIdentities(&getProfileIdentitiesOptions)
```